### PR TITLE
Verify TC_06_25_01 | Move folder to folder

### PR DESCRIPTION
**Pull request by LenaDik**
Verify TC_06_25_01 | Move folder to folder

### CODE REVIEW

File: page_objects/documentsPage.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ class DocumentsPage {
         getYesDeteleBtn: () => this.page.getByRole('button', {name: 'Yes, Delete'}),
         getRenameBtn: () => this.page.getByRole('button', {name: 'Rename'}),
         getRenameInputField: () => this.page.locator('input.form__input--hidden'),
+        getToaster: () => this.page.getByRole('alert'),
```
Comment (verafes) [inline page_objects/documentsPage.js:30]: Please use  'getToast' instead of 'getToaster". Thank you

Comment (LenaDik) [reply page_objects/documentsPage.js:30]: it is used by another tester with a CSS locator ('getToast'), but I use it is used a native locator ('getToaster")

Comment (LenaDik) [reply page_objects/documentsPage.js:30]: getModalWindow?

Comment (LenaDik) [reply page_objects/documentsPage.js]: I changed

**Review by verafes — APPROVED** [review]

**Pull request closed without merging**